recipe/meta.yaml

```diff
@@ -50,8 +41,10 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
     - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    - cf-nvidia-tools                       # [linux]
+    # https://github.com/NixOS/patchelf/issues/492
+    - patchelf >=0.12,<0.18.0               # [linux]
```

Comment:
Moved to `patchelf` to `requirements/build` as it is a build tool

Also added an upper bound (as we have done elsewhere) to protect against this issue: https://github.com/NixOS/patchelf/issues/492

Reply:
This should probably be a dependency of cf-nvidia-tools instead.

Reply:
Until the logic using `patchelf` moves ( https://github.com/conda-forge/cf-nvidia-tools-feedstock/issues/1 ), I don't think that makes sense

Reply:
https://github.com/conda-forge/cf-nvidia-tools-feedstock/pull/3

Reply:
This recipe doesn't use patchelf during install, so the upper bound is not necessary nor is patchelf itself.
```suggestion
```

Reply:
Good point. I wonder why it was here in the first place then

https://github.com/conda-forge/cutensor-feedstock/blob/cb7db90d27df1dcd6fb926f27e69ecfdb09a53f5/recipe/meta.yaml#L53-L54

Reply:
Dunno.

Reply:
It looks like there was a hairy bug Leo was fixing: https://github.com/conda-forge/cutensor-feedstock/issues/16

Think this sort of thing is better handled by adding it `remote_ci_setup` in `conda-forge.yml`. So will move it there

Reply:
Ah right we can't do that either: https://github.com/conda-forge/conda-smithy/issues/1737#issuecomment-1614409419

Nvm `patchelf` has been upgraded several times since `0.12.0`. Current version used in conda-forge is `0.17.2` (`0.18.0` was marked broken due to another bug)

Given this think we don't need this constraint here